libr/core/graph.c

```diff
@@ -3902,15 +3904,10 @@ R_API int r_core_visual_graph(RCore *core, RAGraph *g, RAnalFunction *_fcn, int
 			break;
 		case 'O':
 			disMode++;
-			if (disMode > 3) {
+			if (disMode > 2) {
```

Comment:
instead of the ++ and if, i think it's more idiomatic to just do `disMode = (disMode + 1) % 3`